target/linux/ipq806x/files/arch/arm/boot/dts/qcom-ipq8064-c2600.dts

```diff
@@ -36,22 +37,44 @@
 
 	soc {
 		pinmux@800000 {
+			button_pins: button_pins {
+				mux {
+					pins = "gpio16", "gpio54", "gpio65";
+					function = "gpio";
+					drive-strength = <2>;
+					bias-pull-up;
+				};
+			};
+
 			i2c4_pins: i2c4_pinmux {
-				pins = "gpio12", "gpio13";
-				function = "gsbi4";
-				bias-disable;
+				mux {
+					pins = "gpio12", "gpio13";
+					function = "gsbi4";
+					drive-strength = <12>;
+					bias-disable;
+				};
+			};
+
+			led_pins: led_pins {
+				mux {
+					pins = "gpio6", "gpio7", "gpio8", "gpio9", "gpio26", "gpio33",
+					       "gpio53", "gpio66";
+					function = "gpio";
+					drive-strength = <2>;
+					bias-pull-up;
+				};
 			};
 
 			spi_pins: spi_pins {
 				mux {
-					pins = "gpio18", "gpio19", "gpio21";
+					pins = "gpio18", "gpio19", "gpio20", "gpio21";
```

Comment:
So maybe just add gsbi function to line 81 then? :) besides i don't know if it's correct to set gsbi function to chip-select pin (20)